app/lib/features/cross_signing/widgets/cross_signing.dart

```diff
@@ -218,11 +218,12 @@ class CrossSigningState extends ConsumerState<CrossSigning> {
     });
 
     // start request event loop
-    final client = ref.read(alwaysClientProvider);
+    final client = await ref.read(alwaysClientProvider.future);
     client.installRequestEventHandler(fId);
 
     // open verification.request dialog
     showModalBottomSheet(
+      // ignore: use_build_context_synchronously
```

Comment:
this entire code isn't really being used now, so I opted for just ignoring the lint in this case.